_includes/fix_analytics_signup_form.html

```diff
@@ -0,0 +1,412 @@
+                <span>
+                    {% if lang_da %}
+                        Må vi kontakte jer med henblik på at profilere jer som en privacy-bevidst virksomhed i branding relateret til Fix Analytics-produktet?
+                    {% else %}
+                        Can we contact you for the purpose of profiling you as a privacy-conscious company in branding related to the Fix Analytics product?
+                    {% endif %}
+                </span>
+                <input
+                    type="checkbox"
+                    name="contactForProfiling"
+                />
+            </label>
+            <label class="checkbox-container">
+                <span>
+                    {% if lang_da %}
+                        Jeg accepterer <a target="_blank" href="/fix-google-analytics/vilkaar">vilkår og databehandleraftale</a>
+                    {% else %}
+                        I accept the <a target="_blank" href="/fix-google-analytics/vilkaar">terms and conditions</a>
+                    {% endif %}
+                </span>
+                <input
+                    required
+                    type="checkbox"
+                    name="accept_terms"
+                />
+            </label>
+            <button type="submit">{% if lang_da %} Tilmeld {% else %} Signup {% endif %}</button>
+            <div
+                class="contact-form-overlay contact-form-overlay-clickthrough"
+                data-analytics-form-overlay
+            >
+                <div class="contact-form-overlay__message contact-form-overlay__message--awaiting">
+                    {% if lang_da %}
+                        Sender...
+                    {% else %}
+                        Sending request...
+                    {% endif %}
+                </div>
+                <div class="contact-form-overlay__message contact-form-overlay__message--success">
+                    <p>
+                        {% if lang_da %}
+                            Succes! 👍<br>
+                            Du modtager en mail med information omkring implementation.
+                        {% else %}
+                            Success! 👍<br>
+                            Please confirm you have recieved an email and your order has been registered.
+                        {% endif %}
+                    </p>
+                </div>
+                <div class="contact-form-overlay__message contact-form-overlay__message--error">
+                    <p>
+                        {% if lang_da %}
+                            Noget gik desværre galt - Prøv igen ❌
+                        {% else %}
+                            Something went wrong - try again! ❌
+                        {% endif %}
+                    </p>
+                </div>
+            </div>
+        </div>
+    </form>
+</section>
+
+<script>
+    (function () {
+        var form = document.getElementById("contact-form-analytics");
```

Comment:
Here you can use `document.forms['contact-form-analytics']` which is more performant